.nb-gradle-properties

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<gradle-project-properties>
+  <!--DO NOT EDIT THIS FILE! - Used by the Gradle plugin of NetBeans.-->
+  <common-tasks>
+    <task>
+      <display-name>Debug Frodos</display-name>
+      <non-blocking>no</non-blocking>
+      <task-names>
+        <name must-exist="no">installFrodos</name>
+        <name must-exist="no">--stacktrace</name>
+        <name must-exist="no">--info</name>
+      </task-names>
+      <task-args/>
+      <task-jvm-args/>
+    </task>
+  </common-tasks>
+</gradle-project-properties>
```

Comment:
This file needs to be excluded from the PR, actually you can add this file to the .gitignore file as well to be safe.